LEAF_Request_Portal/admin/templates/mod_templates.tpl

```diff
@@ -50,247 +360,545 @@
-             dialog_message.indicateIdle();
-             dialog_message.show();
-         },
-         fail: function() {
-             dialog_message.setContent('Loading failed.');
-             dialog_message.show();
-         },
-         cache: false
-     });
- }
-
-var codeEditor = null;
-$(function() {
-	dialog = new dialogController('confirm_xhrDialog', 'confirm_xhr', 'confirm_loadIndicator', 'confirm_button_save', 'confirm_button_cancelchange');
-
-    initEditor();
-
-    $(window).on('resize', function() {
+        });
+    }
+
+    function editorExpandScreen() {
+        $('.page-title-container>.file_replace_file_btn').show();
+        $('.page-title-container>.close_expand_mode_screen').show();
+        $('.page-title-container>h2').css({
+            'width': '35%',
+            'text-align': 'left'
+        });
+        $('.main-content').css({
+            'width': '100%',
+            'height': '80%',
+            'top': 0,
+            'left': 0,
+            'align-items': 'center',
+            'transition': 'all 1s ease'
+        });
+        $('.leaf-code-container').css({
+            'width': '100% !important'
+        });
+        $('.usa-table').hide();
+        $('.leaf-right-nav').css({
+            'position': 'fixed',
+            'right': '-100%',
+            'transition': 'all 3s ease'
+        });
+        $('.leaf-left-nav').css({
+            'position': 'fixed',
+            'left': '-100%',
+            'transition': 'all 3s ease'
+        });
+        $('.page-title-container').css({
+            'flex-direction': 'coloumn'
+        });
+    }
+
+    function exitExpandScreen() {
+        $('#word-wrap-button').hide();
+        $('.page-title-container>.file_replace_file_btn').hide();
+        $('.page-title-container>.close_expand_mode_screen').hide();
+        $('.page-title-container>h2').css({
+            'width': '100%',
+            'text-align': 'center'
+        });
+        $('.main-content').css({
+            'width': '60%',
+            'height': '80%',
+            'top': 0,
+            'left': 0,
+            'align-items': 'center',
+            'transition': 'all 1s ease'
+        });
+        $('#codeContainer').css({
+            'height': '95%',
+            'width': '90% !important'
+        })
+        $('.usa-table').show();
+        $('.leaf-right-nav').css({
+            'position': 'relative',
+            'right': '0',
+            'transition': 'all 1.5s ease'
+        });
+        $('.leaf-left-nav').css({
+            'position': 'relative',
+            'left': '0',
+            'transition': 'all 1.5s ease'
+        });
+        $('.page-title-container').css({
+            'flex-direction': 'row'
+        });
+
+        $('#save_button').css('display', 'block');
+        loadContent();
+    }
+
+
+    var currentFile = '';
+    var currentFileContent = '';
+
+    function loadContent(file) {
+        if (file == undefined) {
+            file = currentFile;
```

Comment:
May be worthwhile to alert the user that the file could not be loaded.

Reply:
Will add that, thank you!